openpmd_viewer/openpmd_timeseries/data_reader/data_reader.py

```diff
@@ -49,11 +46,7 @@ def __init__(self, backend_set):
         """
```

Comment:
may be we can change argument name `backend_set` to simply `backend` since its not a module attribute any more

Reply:
it is - see on the right :)